### ISSUE TITLE
File explorer crashing on startup

### ISSUE THREAD

**Issue by megagigawatt** [valinet/ExplorerPatcher#1814]
File explorer continually crashing on startup with the latest windows build Version 10.0.22621 Build 22621

Explorer patcher disables itself after 3 crashes but startup apps aren't loading because of the crashing

**Comment by megagigawatt** [comment]
> The issues were resolved on my Windows 11 after deleting the keys `IsUndockedAssetAvailable` (located at `HKEY_CURRENT_USER\Software\ExplorerPatcher\twinui.pcshell`) and `OSBuild` (located at `HKEY_CURRENT_USER\Software\ExplorerPatcher`). After that, I restarted EP and about 3 minutes later I received a notification of symbols being downloaded. After that, I restarted EP again and had no more problems.

Awesome, I will give this a try, I actually just uninstalled it for now, so will get it reinstalled and see if it happens, if it does will try this

**Comment by alanenggb** [comment]
> The issues were resolved on my Windows 11 after deleting the keys IsUndockedAssetAvailable (located at HKEY_CURRENT_USER\Software\ExplorerPatcher\twinui.pcshell) and OSBuild (located at HKEY_CURRENT_USER\Software\ExplorerPatcher). After that, I restarted EP and about 3 minutes later I received a notification of symbols being downloaded. After that, I restarted EP again and had no more problems.

I did this and it stopped crashing and downloaded the new version, but now the win+tab and the task view button are not opening the Task View. I still can switch desktops with ctrl+win+arrows though.

**Comment by toufou54** [comment]
Same, corrected the crash and ALT + TAB worked only once after that

**Comment by skydestro** [comment]
> File explorer continually crashing on startup with the latest windows build Version 10.0.22621 Build 22621
> 
> Explorer patcher disables itself after 3 crashes but startup apps aren't loading because of the crashing

yeah I get this issue now too after updating my windows. Don't wana try the fixes shown above cus of the key shortcuts not working mentioned.

**Comment by Rolzzz** [comment]
crashing here for me too!!

**Comment by qheartq616** [comment]
crashing here too
22621.2134

**Comment by Amrsatrio** [comment]
Crashing on me too, 22621.2134

**Comment by Iory1998** [comment]
Hello, I am facing similar issue. For so long, I have been facing with the wifi icon not working. No that the ExplorerPatcher being disabled, I can have access to the wifi icon on the taskbar.

**Comment by sybaris** [comment]
Crashing on me too, on windows 11 : 22621.2134
![image](https://github.com/valinet/ExplorerPatcher/assets/11009997/23d99390-509c-4515-9144-ba7fd0fedaf6)

**Comment by Majoraslayer** [comment]
Also crashing for me after Windows Update.  Preview builds are disabled per advice from the last time this happened recently, so it might be helpful to look into those as a preventative measure for things to come in official update releases.

**Comment by Rolzzz** [comment]
> Also crashing for me after Windows Update. Preview builds are disabled per advice from the last time this happened recently, so it might be helpful to look into those as a preventative measure for things to come in official update releases.

Yeah not running preview builds on my daily driver… will try reg edit options and revert back

**Comment by dazednova** [comment]
same thing here immediately after updating windows to windows 11 home os build 22621.2134, i'll disable it for now as i don't need the program that badly and it's evidently not just me having the issue

**Comment by jollyfish91** [comment]
or you can just revert the update unistall explorer patcher and then restart the pc and update it again.
 And still having the property window  even tho you unistalled explorer patcher aahahhahaahah
![Immagine 2023-08-11 135337](https://github.com/valinet/ExplorerPatcher/assets/142010268/6bd73440-3b95-4c1b-aff0-1c0715523027)

**Comment by jollyfish91** [comment]
![Immagine 2023-08-11 135531](https://github.com/valinet/ExplorerPatcher/assets/142010268/eafa1231-7247-4956-aa22-d389d2c35c23)
![todd-howard-it-just-works](https://github.com/valinet/ExplorerPatcher/assets/142010268/d9ea043a-1c95-4f79-a14b-39bf043074ba)

**Comment by regs01** [comment]
Once updated symbols it's no longer crashing. No workarounds needed.

**Comment by Majoraslayer** [comment]
> Once updated symbols it's no longer crashing. No workarounds needed.

I'm admittedly a bit ignorant on what the "symbols" actually are, though I've seen it update them.  Is a set of them included with a fresh installation of the program, or does a fresh install have to download them?  If the latter, a quick fix without temporarily reverting Windows Updates seems like could be done by downloading an updated installer (or even an ExplorerPatcher.....Patcher).  I've not tried reverting Windows and forcing an ExplorerPatcher update to see if it downloads new symbols yet to fix it, but I am curious to more streamlined alternatives for fixing it if this is going to continue being a problem with Windows updates.

**Comment by Stuwot** [comment]
> Once updated symbols it's no longer crashing. No workarounds needed.

Just logged into say the same.  Windows update ran, then Explorer started crashing.  I downloaded the latest version of EP and ran the installer and even then it continued to crash - but then after a second attempt at installing the latest EP, it told me that it had gotten the updated symbols (whatever they are lol) and everything seems to be working now.

**Comment by alanenggb** [comment]
I tried to uninstall then install again the latest version, the alt+tab works until I try to open the Task View with win+tab, then alt+tab stops working

**Comment by pyrates999** [comment]
Is this happening because symbols are not downloaded yet?  If so, read here: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

This kind of bug gets reported every time on the second Tuesday of every month due to symbols not being downloaded yet for the new update that was just released.

**Comment by megagigawatt** [comment]
I am still getting issues even with trying the various recommended fixes, will wait till EP gets updated and then install it, will just suffer a couple of weeks without

**Comment by qheartq616** [comment]
It works. Cool.
Last time the same things happend, I forced it updated symbols by open its console (properties-advanced-enable console). It worked. But this time when I open the console, there was a bug and ep didn't update symbols automatically.

**Comment by pyrates999** [comment]
Can you show the results of that console output where you said there was a bug?

**Comment by regs01** [comment]
> I'm admittedly a bit ignorant on what the "symbols" actually are

Files containing information about compiled libraries - functions and their locations, variables and so on.

**Comment by regs01** [comment]
One thing - volume popup doesn't work.

**Comment by pyrates999** [comment]
> One thing - volume popup doesn't work.

https://github.com/valinet/ExplorerPatcher/issues/1692

**Comment by Poopooracoocoo** [comment]
Duplicate of #1798
